nemo/collections/nlp/models/language_modeling/megatron_gpt_model.py

```diff
@@ -1722,6 +1724,14 @@ def sharded_state_dict(self, prefix: str = '') -> Dict[str, Any]:
             if parallel_state.get_virtual_pipeline_model_parallel_world_size() is not None:
                 parallel_state.set_virtual_pipeline_model_parallel_rank(0)
 
+            def skip_fp8_load(x):
```

Comment:
@rachitgarg91 Is this intended as a permanent fix?
If so, this should end up in the attention module or at least a TransformerLayer.
If it's temporary (I hope so?) please leave a comment that it's a WAR

Reply:
also, does it make sense to move it to MCore?